src/cli/index.js

```diff
@@ -171,6 +181,11 @@ export function main({
   //
   const run = (): Promise<void> => {
     invariant(command, 'missing command');
+
+    if (warnAboutRunDashDash) {
+      reporter.warn(reporter.lang('dashDashDeprecation'));
```

Comment:
Makes sense, but the PR that will implement the `--strict` feature will need to revisit all the loglevels anyway.